packages/patrol_finders/test/utils/screen_with_long_button.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class ScreenWithLongButton extends StatelessWidget {
```

Comment:
The name is misleading, we are not always passing button. How about `ScreenWithPartiallyVisibleWidget`?